Browser build failure on Windows

After patching to fix the FromConstructorTemplate link error (#108), I'm seeing an "Error: Command failed: The system cannot find the path specified" error in the browser/build.js segment of the bitcore build.

```
npm build bitcore
[snip]
> bitcore@0.1.11 postinstall C:\Users\dthorpe\Documents\bitcore
> node browser/build.js -a

Including Address in the browser bundle
Including BIP32 in the browser bundle
Including Block in the browser bundle
Including Bloom in the browser bundle
Including Buffers.monkey in the browser bundle
Including Connection in the browser bundle
Including Deserialize in the browser bundle
Including Gruntfile in the browser bundle
Including Number.monkey in the browser bundle
Including Opcode in the browser bundle
Including Peer in the browser bundle
Including PeerManager in the browser bundle
Including PrivateKey in the browser bundle
Including RpcClient in the browser bundle
Including Key in the browser bundle
Including Point in the browser bundle
Including SIN in the browser bundle
Including SINKey in the browser bundle
Including Script in the browser bundle
Including ScriptInterpreter in the browser bundle
Including Sign in the browser bundle
Including Transaction in the browser bundle
Including TransactionBuilder in the browser bundle
Including Wallet in the browser bundle
Including WalletKey in the browser bundle
Including config in the browser bundle
Including const in the browser bundle
Including networks in the browser bundle
Including util/log in the browser bundle
Including util/util in the browser bundle
Including util/EncodedData in the browser bundle
Including util/VersionedData in the browser bundle
Including util/BinaryParser in the browser bundle
{ [Error: Command failed: The system cannot find the path specified.
] killed: false, code: 1, signal: null }

stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
Error: module "./browser/vendor-bundle.js" not found from "C:\\Users\\dthorpe\\D
ocuments\\bitcore\\Key.js"
    at notFound (C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\inde
x.js:744:15)
    at C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\index.js:695:2
3
    at C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\node_modules\b
rowser-resolve\index.js:183:24
    at C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\node_modules\b
rowser-resolve\node_modules\resolve\lib\async.js:36:22
    at load (C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\node_mod
ules\browser-resolve\node_modules\resolve\lib\async.js:54:43)
    at C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\node_modules\b
rowser-resolve\node_modules\resolve\lib\async.js:60:22
    at C:\Users\dthorpe\Documents\bitcore\node_modules\browserify\node_modules\b
rowser-resolve\node_modules\resolve\lib\async.js:16:47
    at Object.oncomplete (fs.js:107:15)

npm ERR! bitcore@0.1.11 postinstall: `node browser/build.js -a`
npm ERR! Exit status 8
```

The build did create a file bitcoin/browser/bundle.js but it is zero length.

--- a/browser/build.js
+++ b/browser/build.js
@@ -63,7 +63,10 @@ var createBitcore = function(opts) {
   opts.dir = opts.dir || '';
 
   // concat browser vendor files
-  exec('cd ' + opts.dir + 'browser & sh concat.sh', puts);
+  var cwd = process.cwd();
+  process.chdir(opts.dir + 'browser');
+  exec('sh concat.sh', puts);
+  process.chdir(cwd);
 
   if (!opts.includeall && (!opts.submodules || opts.submodules.length === 0)) {
     if (!opts.stdout) console.log('Must use either -s or -a option. For more info use the --help option');

